### DISCUSSION TITLE
Can't find Remote Apps in TeamSpeak 3

### DISCUSSION THREAD

**Discussion by CobbleShop** [LeonMarcel-HD/StreamDeck-TeamSpeak5#4] [Q&A / Help] [answered]
Hey. I don't have the "Remote Apps" option in the settings menu. 
Could you guide me on how to find it?

**Comment by LeonMarcel-HD** [comment, marked as answer]
# ℹ️  This Plugin only works with TeamSpeak 5. 
You can download TeamSpeak 5 here. https://www.teamspeak.com/en/downloads/#ts5 and you will then be able to use the Plugin.


Here is a guide on where to find the Remote Apps in the TeamSpeak 5 settings. 

![TeamSpeak_zCP9fdQ3xF](https://github.com/LeonMarcel-HD/StreamDeck-TeamSpeak5/assets/83507548/e6247873-d579-4226-b70a-ecf61d848d23)

**Comment by CobbleShop** [comment]
I got 3.6.0 and was told that this is "TeamSpeak5", right?


Am Fr., 7. Juli 2023 um 22:21 Uhr schrieb Leon ***@***.***>:

**Comment by LeonMarcel-HD** [comment]
No 3.6.0 is TeamSpeak 3. You can download TeamSpeak 5 here. https://www.teamspeak.com/en/downloads/#ts5

Currently you still need a myTeamSpeak Account to access the Client as it is still in Beta.
